Source/WebKit/Shared/WebMouseEvent.h

```diff
@@ -43,7 +43,8 @@ namespace WebKit {
 
 enum class GestureWasCancelled : bool { No, Yes };
 
-enum class WebMouseEventButton : int32_t {
+// FIXME [aprotyas]: make this an enum class with int8_t backing type as well
```

Comment:
Nit - we generally do `FIXME (Bugzilla number): …` instead of `FIXME (username): …` in WebKit.

Reply:
Yikes good catch, this comment is (a) wrong and (b) outdated... not sure how I committed it